bot/es-ES/commands.json

```
@@ -595,18 +595,18 @@
       "sureDesc": "`Su suscripción de **{{amount}}** será cancelada **inmediatamente** y no se procesará su próximo pago. Su suscripción a premium acabará el **{{date}}**.",
       "done": "Se ha cancelado su suscripción correctamente, esperamos volver a verle. :(\nSu suscripción acabará el **{{date}}**.",
       "selectTitle": "Selección de suscripción",
-      "selectDesc": "Como tienes varias suscripciones activas debe seleccionar la que desea cancelar.",
+      "selectDesc": "Como tiene varias suscripciones activas debe seleccionar la que desea cancelar.",
       "selectPlaceholder": "Seleccione una suscripción"
     }
   },
   "reason": {
-    "noCase": "**Error**, no se ha encontrado el caso, comprueba que el ID del caso sea correcto.",
+    "noCase": "**Error**, no se ha encontrado el caso, compruebe que la ID del caso sea correcto.",
```

```suggestion
    "noCase": "**Error**, no se ha encontrado el caso, compruebe que el ID del caso sea correcto.",
```

ID se refiere a identificador, es masculino